src/components/common/DynamicTabs/DynamicTabs.scss

```diff
@@ -12,7 +12,7 @@
     .more-tabs-wrapper {
         .more-tabs__menu-wrapper {
             border-radius: 4px 4px 0 0;
-            left: 8px;
+            left: -260px;
```

Comment:
Is this left value expected? Can we fix parent wrapper or container width instead if required?